xla/service/latency_hiding_scheduler.h

```diff
@@ -139,6 +139,7 @@ struct SchedulerConfig {
   bool enable_selective_resources = false;
   int64_t max_hops_to_closest_selective_overlap = 0;
   int64_t rerun = 0;
+  int64_t global_collective_scheduling_resource = 1;
```

Comment:
Maybe change to `parallel_collective_overlap_limit`?

Reply:
yea sounds good